ext/pdo_sqlite/tests/subclasses/pdosqlite_002.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+PdoSqlite create through PDO::connect and function define.
+--EXTENSIONS--
+pdo_sqlite
+--FILE--
+<?php
+
+$db = Pdo::connect('sqlite::memory:');
+if (!$db instanceof PdoSqlite) {
+    echo "Wrong class type. Should be PdoSqlite but is " . get_class($db) . "\n";
+}
+
+$db->query('CREATE TABLE IF NOT EXISTS pdosqlite_002 (id INT AUTO INCREMENT, name TEXT)');
```

Comment:
```suggestion
$db->query('CREATE TABLE pdosqlite_002 (id INT AUTO INCREMENT, name TEXT)');
```
I suppose this shouldn't be needed because of the in memory DB?

Reply:
Yeah, right